src/features/currentTodo.ts

```diff
@@ -1,10 +1,14 @@
-import { createSlice } from '@reduxjs/toolkit';
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
 import { Todo } from '../types/Todo';
 
 const initialState = null as Todo | null;
 
 export const currentTodoSlice = createSlice({
   name: 'currentTodo',
   initialState,
-  reducers: {},
+  reducers: {
+    changecurrentTodo(_currentTodo, { payload }: PayloadAction<Todo | null>) {
```

Comment:
There is a typo in the reducer name `changecurrentTodo`. It should be `changeCurrentTodo` to follow the camelCase naming convention and ensure consistency across the codebase.